src/lsdb/loaders/dataframe/read_dataframe.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from hipscat.catalog import CatalogType
+
+from lsdb.catalog.dataset.dataset import Dataset
+from lsdb.loaders.dataframe.dataframe_catalog_loader import DataframeCatalogLoader
+from lsdb.loaders.hipscat.hipscat_loader_factory import CatalogTypeVar
+
+
+def read_dataframe(
```

Comment:
Ah naming. So I think we're trying to match the pandas/dask convention for our loading methods. They use `read_...` for anything that's being read from disk, and `from_...` for anything that's being made from an object in memory. This is currently reading a csv file, but I think the desired version of this function is one that uses a Pandas Dataframe as an input, so that should be named `from_dataframe`